test/assets.js

```diff
@@ -21,4 +89,60 @@ describe('Assets', function() {
     const text = await apos.http.get('/static-test.txt');
     assert(text.match(/served/));
   });
+
+  it('should check that webpack configs in modules are well formatted', async function () {
+    const [ valid, err ] = check(apos.modules, apos.task.getReq().t);
+
+    assert(valid);
+    assert(!err);
+
+    await t.destroy(apos);
+
+    apos = await t.create({
+      root: module,
+      modules: badModules
+    });
+
+    const [ valid2, err2 ] = check(apos.modules, apos.task.getReq().t);
+
+    assert(!valid2);
+    assert(err2);
+
+    await t.destroy(apos);
+
+    apos = await t.create({
+      root: module,
+      modules
+    });
+
+    function check (modules, t) {
+      try {
+        checkModulesWebpackConfig(modules, t);
+
+        return [ true, false ];
+      } catch (err) {
+        return [ false, err ];
+      }
+    }
+  });
+
+  it('shoud merge webpack configs from modules with the Apostrophe one', async function () {
```

Comment:
The unit tests must show that if a base class and a subclass both contribute webpack extensions with different names, they both make it through, even if the base class module is never instantiated on its own.

This is similar to the way the `i18n` section allows base classes and subclasses both to contribute without conflicting with each other.